package.json

```diff
@@ -30,7 +30,8 @@
     "prepack": "pnpm build",
     "release": "pnpm test && changelogen --release && npm publish && git push --follow-tags && pnpm build --stub",
     "test": "pnpm lint && pnpm test:types && vitest run --coverage",
-    "test:types": "tsc --noEmit --skipLibCheck"
+    "test:types": "tsc --noEmit --skipLibCheck",
+    "test:generators": "vitest run test/generators"
```

Comment:
Can you please extend existing tests? 🙏🏼 

https://github.com/unjs/automd/tree/main/test/fixture